_includes/setup/path-win.md

```diff
@@ -0,0 +1,23 @@
+### Update your path
```

Comment:
OK tested this, and read some stuff on the Internet. The way we have the syntax, it just updates the USER variable. Not SYSTEM. Some users will not be able to use setx to update SYSTEM PATH (I wasn't allowed). So, leaving this section as-is.